rasa/importers/importer.py

```diff
@@ -368,19 +374,21 @@ def _message_from_conversation_event(event: Event) -> Optional[Message]:
 
 
 def _messages_from_user_utterance(event: UserUttered) -> Message:
-    return Message(event.text, data={MESSAGE_INTENT_NAME: event.intent_name})
+    return Message(event.text, data={INTENT_NAME: event.intent_name})
 
 
 def _messages_from_action(event: ActionExecuted) -> Message:
-    return Message(event.e2e_text or "", data={MESSAGE_ACTION_NAME: event.action_name})
+    return Message(
+        "", data={ACTION_NAME: event.action_name, ACTION_TEXT: event.e2e_text or "",},
+    )
```

Comment:
there is `Message.build` method that creates message with text, intents and entities, let's create a similar `build_for_actions` or smth that would create message from action_name and action_text

Reply:
also I think `Message.text` doesn't make sense anymore